pkg/common/utils/utils.go

```diff
@@ -210,9 +210,9 @@ func QuerySnapshotsUtil(ctx context.Context, m cnsvolume.Manager, snapshotQueryF
 	return allQuerySnapshotResults, "", nil
 }
 
-// Query Capacity in MB for block volume snapshot
-func QueryCapacityForBlockVolumeSnapshotUtil(ctx context.Context, m cnsvolume.Manager, sourceVolumeID,
-	blockVolumeType string) (int64, error) {
+// Query Capacity in MB and datastore URL for the source volume with expected volume type
+func QueryCapacityAndDsUrlForVolumeUtil(ctx context.Context, m cnsvolume.Manager, sourceVolumeID,
+	volumeType string) (snapshotSizeInMB int64, datastoreUrl string, queryErr error) {
```

Comment:
Any reason you are using named return? most of the other csi driver code doesn't follow this format.

Reply:
Basically, It is more self-explanatory about what the return values are. So, we don't need to write extra comments for the output of method.

Reply:
I'd prefer to keep the previous return style to be consistent with other code. Also it's easier to see what's get returned that way.